crawl-ref/source/mutation.cc

```diff
@@ -97,62 +97,54 @@ static const body_facet_def _body_facets[] =
     { EQ_BOOTS, MUT_TALONS, 3 }
 };
 
+enum mutation_resolution
+{
+    MUTR_REPLACE, // Remove the existing mutation and replace with the new one.
+    MUTR_COEXIST, // Allow both mutations to exist.
+    MUTR_MODIFY,  // Delete a level of the existing mutation.
+    MUTR_IGNORE,  // Discard the new mutation, leaving existing mutations unchanged.
+};
```

Comment:
This approach doesn't make sense. override isn't an arbitrary boolean toggle between two modes, it's a signal to attempt to add the mutation even when it'd be normally impossible. Splitting the 0/1/-1 values into two mutation_resolutions gives you the flexibility to say things like "replace existing mutations, UNLESS override is set, in which case you should just give up (MUTR_IGNORE) instead!" Not only do you not need that kind of flexibility for existing interactions, it doesn't make any sense.

I'd recommend just replacing the -1/0/1 values with an enum.